chainercv/links/__init__.py

```diff
@@ -1,3 +1,5 @@
+from chainercv.links.connection.conv_2d_activ import Conv2DActiv
```

Comment:
I made it to be consistent with the Chainer's directory structure.
`connection` seems to include links with learnable parameters (e.g. Convolution2D) and a block of them (e.g. Inception).
If we change the directory structure, we need to come up with a new name for group of links that are not blocks, but categorized under `connection`.